libraries/apollo-server/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "@prisma/client": "3.7.0-dev.31",
-    "@types/node-fetch": "2.5.12",
+    "@types/node-fetch": "3.0.2",
```

Comment:
Why types as a main dependency, and not the actual package? Does this make sense?